base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleData.java

```diff
@@ -1,793 +1,854 @@
+    /**
+     * @since SmartDeviceLink 7.1.0
+     */
+    public static final String KEY_CLIMATE_DATA = "climateData";
+
+    /**
+     * Constructs a new GetVehicleData object
+     */
+
+    public GetVehicleData() {
+        super(FunctionID.GET_VEHICLE_DATA.toString());
+    }
+
+    /**
+     * <p>
+     * Constructs a new GetVehicleDta object indicated by the Hashtable
+     * parameter
+     * </p>
+     *
+     * @param hash The Hashtable to use
+     */
+
+    public GetVehicleData(Hashtable<String, Object> hash) {
+        super(hash);
+    }
+
+    public GetVehicleData setGps(Boolean gps) {
+        setParameters(KEY_GPS, gps);
+        return this;
+    }
+
+    public Boolean getGps() {
+        return getBoolean(KEY_GPS);
+    }
+
+    public GetVehicleData setSpeed(Boolean speed) {
+        setParameters(KEY_SPEED, speed);
+        return this;
+    }
+
+    public Boolean getSpeed() {
+        return getBoolean(KEY_SPEED);
+    }
+
+    public GetVehicleData setRpm(Boolean rpm) {
+        setParameters(KEY_RPM, rpm);
+        return this;
+    }
+
+    public Boolean getRpm() {
+        return getBoolean(KEY_RPM);
+    }
+
+    /**
+     * Sets the fuelLevel.
+     *
+     * @param fuelLevel The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec
+     *                  7.0, please see fuelRange.
+     */
+    @Deprecated
+    public GetVehicleData setFuelLevel(Boolean fuelLevel) {
+        setParameters(KEY_FUEL_LEVEL, fuelLevel);
+        return this;
+    }
+
+    /**
+     * Gets the fuelLevel.
+     *
+     * @return Boolean The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec
+     * 7.0, please see fuelRange.
+     */
+    @Deprecated
+    public Boolean getFuelLevel() {
+        return getBoolean(KEY_FUEL_LEVEL);
+    }
+
+    @Deprecated
+    public GetVehicleData setFuelLevelState(Boolean fuelLevelState) {
+        setParameters(KEY_FUEL_LEVEL_STATE, fuelLevelState);
+        return this;
+    }
+
+    @Deprecated
+    public Boolean getFuelLevelState() {
+        return getBoolean(KEY_FUEL_LEVEL_STATE);
+    }
+
+    public GetVehicleData setInstantFuelConsumption(Boolean instantFuelConsumption) {
+        setParameters(KEY_INSTANT_FUEL_CONSUMPTION, instantFuelConsumption);
+        return this;
+    }
+
+    public Boolean getInstantFuelConsumption() {
+        return getBoolean(KEY_INSTANT_FUEL_CONSUMPTION);
+    }
+
+    public GetVehicleData setFuelRange(Boolean fuelRange) {
+        setParameters(KEY_FUEL_RANGE, fuelRange);
+        return this;
+    }
+
+    public Boolean getFuelRange() {
+        return getBoolean(KEY_FUEL_RANGE);
+    }
+
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
```

Comment:
@kostyaBoss On line 464, when using the RPC generator tool, I got 7.1.0.  I would replace 7.1 with 7.1.0.